app/api/account.py

```diff
@@ -25,3 +33,18 @@ async def get_accounts(token: str = Depends(get_current_user)):
 async def get_account(account_id: int, token: str = Depends(get_current_user)):
     results = await read_account(account_id, token)
     return results
```

Comment:
P2: 존재하지 않는 account_id로 요청이 들어오는 경우도 처리해야하므로 해당 케이스에서는 not found를 돌려주는 것이 좋을 것 같습니다. 물론 서비스쪽에서도 DB에서 account_id로 특정 가계부를 찾을 수 없으면 None을 반환한다는 가정하에요.
```suggestion
    account = await read_account(account_id, token)
    if account is None:
        raise account_not_found() # 별도로 정의한다고 가정..
    return account
```

Reply:
수정했습니다. 
c6b0952